src/ibrowse_http_client.erl

```diff
@@ -465,7 +465,7 @@ accumulate_response(Data, #state{reply_buffer      = RepBuf,
 
 make_tmp_filename(true) ->
     DownloadDir = ibrowse:get_config_value(download_dir, filename:absname("./")),
-    {A,B,C} = now(),
+    {A,B,C} = {erlang:unique_integer([positive, monotonic]), 0, 0},
```

Comment:
Good call. Here it also expects it to be strictly monotonic. But seeing that this creates a file made think of another case when the erlang VM is restarted. In that case the counter will be reset to 0 and so filename might collide. I think we might want to keep at least some connection to wall clock time here.

Perhaps something like:

```
strictly_monotonic_timestamp() ->
   {A, B, C} = erlang:timestamp(),
   Unique = erlang:unique_integer([positive, monotonic]),
   {A, B, C + Unique}.
```